### ISSUE TITLE
Views are sorted alphabetically, not in file order

### ISSUE THREAD

**Issue by mattfahrner** [thias/puppet-bind#70]
In your example of how to use views you give 'trusted' followed by 'default', however in practice the view names get sorted alphabetically, so 'default' will be output before 'trusted'. Since 'default' has catch all "match-clients" of "any", the 'trusted' won't trigger as first catch all 'default' match will always win out.

Changing the view name to something like 'a_trusted' will make it sort prior and fix the issue.

Note you also put in the same example code a 'zones' section, which I don't believe is allowed - if you use 'views', all zones have to be inside views.

**Comment by landrypm** [comment]
Good catch on the ordering of the views in the resulting named.conf file.
The inclusion of the zone definition outside of the views definitions in the example is fine since this module will place all such zones (defined outside of a view) inside all views.

**Comment by mattfahrner** [comment]
Thanks!
